reactioncommerce/reaction
# (feat): Create GraphQL account query

Resolves #3912 
Impact: **minor**  
Type: **feature**

## Issue
Create an `account` query in GraphQL.

## Solution
Create a query with one argument, `id`, which is required, and return an Account associated with this ID, if the user calling the query has the correct permissions.

## Breaking changes
None

## Testing
1. Start up Reaction
2. Open Mongo (Robo 3T), and copy the `id` of any user
3. Use `Terminal` to Base64 encode the ID into base64 along with an `account` prefix - `echo account:{id} | base64` - and copy the encoded value
4. Navigate to `http://localhost:3000/graphiql`
5. Query to find an account from this encoded ID
```
{
   account(id: {encodedId})
}
```
6. Change the query to an incorrect ID, and see you get a "No account found" Error message

Docs: https://github.com/reactioncommerce/reaction-docs/pull/559
